schemas/account.js

```diff
@@ -10,6 +10,39 @@ const postAccounts = {
   }
 };
 
+const Pagination = {
```

Comment:
Rename, and why capitalized?  Capped names are usually classes.

Reply:
ok cool